pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>net.ripe.db</groupId>
     <artifactId>whois</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>utc-switch-version-2</version>
```

Comment:
Please revert this change before merging

Reply:
Noted. This branch has been merged to `switching-to-utc` which should contain all changes now. And yes, it will be reverted to the right version before merging to master